app/packages/packaged-app/src/server/lib/SettingsServer.coffee

```diff
@@ -1,5 +1,9 @@
 log = new ObjectLogger('server.hubapp_user_settings', 'debug')
 
+github = hubaaa.GitHubApiClient.get()
+syncGetUserRepos = Meteor._wrapAsync github.getUserRepos, github
```

Comment:
hubaaa.GitHubApiClient lib works client/server so having implementation use callbacks is the only way. Right now is not used in the client, but it could in the future. There is also no performance overhead in this particular case. I don't agree we should expend time refactoring this.